src/libpmemfile-posix/dir.c

```diff
@@ -436,6 +436,27 @@ vinode_lookup_dirent_by_name_locked(PMEMfilepool *pfp,
 }
 
 /*
+ * vinode_lookup_vinode_by_name_locked -- looks up file name in passed
+ * directory and returns dirent and vinode
+ */
+struct pmemfile_dirent_info
+vinode_lookup_vinode_by_name_locked(PMEMfilepool *pfp,
```

Comment:
Does it need to have `vinode` in the name twice?
Perhaps it could be `vinode_lookup_dirent_by_name_locked` ?